azext_iot/digitaltwins/commands_resource.py

```diff
@@ -27,7 +28,12 @@ def create_instance(
     scopes=None,
     role_type="Contributor",
     public_network_access=ADTPublicNetworkAccessType.enabled.value,
+    no_wait=False,
 ):
+    if no_wait and scopes:
+        raise CLIError(
```

Comment:
We can use the new CLI error classes here - I believe this would be a `MutuallyExclusiveArgumentError` since a user can either use scopes, or `--no-wait` but not both

Reply:
done